ios/FluentUI/Vnext/List/List.swift

```diff
@@ -57,7 +57,7 @@ public struct MSFListView: View {
                                 .frame(maxWidth: .infinity)
                         }
                         if section.hasDividers {
-                            Divider()
+                            FluentDivider()
                                 .overlay(Color(tokens.borderColor))
```

Comment:
Wasn't sure how we wanted to really handle this. Should I just replace the Divider with the FluentDivider, or should I also remove the .overlay so that we're just using the FluentDivider? Same thing in the ListCell, but then also padding.

Reply:
Assuming that `tokens.borderColor` matches the color for `FluentDivider`, then yeah let's remove the overlay.

As for the `ListCell` padding case, is that padding meant to be additive atop what `FluentDivider` already provides?

Reply:
The padding in the cell is just a leading offset, so it's different. Left it in.